src/github3/checks.py

```diff
@@ -0,0 +1,281 @@
+# -*- coding: utf-8 -*-
+"""This module contains all the classes relating to Checks."""
+from __future__ import unicode_literals
+
+from json import dumps
+
+from . import models
+from .decorators import requires_auth
+
+
+class CheckSuite(models.GitHubCore):
+    """The :class:`CheckSuite <CheckSuite>` object.
+
+    .. versionadded:: 1.2.0
+
+    Please see GitHub's `CheckSuite Documentation`_ for more information.
+
+    .. attribute:: status
+
+        The status of the Check Suite
+
+    .. attribute:: conclusion
+
+        The highest priority check run conclusion. If it has not completed this
+        will be None
+
+    .. attribute:: head_sha
+
+        The sha of the commit at the head of the branch checked
+
+    .. attribute:: head_branch
+
+        The branch checked
+
+    .. attribute:: before
+
+        The sha of the target branch before the change
+
+    .. attribute:: after
+
+        The sha of the target branch after the change is applied
+
+    .. attribute:: repository
+
+        A representation of the repository the suite belongs to as
+        :class:`~github3.repos.repo.ShortRepository`.
+
+    .. attribute:: pull_requests
+
+        A list of representations of the pull requests the suite belongs to as
+        :class:`~github3.pulls.ShortPullRequest`. This may be empty.
+
+    .. attribute:: id
+
+        The unique GitHub assigned numerical id of this check suite.
+
+    .. CheckSuite Documentation:
+        http://developer.github.com/v3/checks/suites/
+    """
+
+    class_name = 'CheckSuite'
+    CUSTOM_HEADERS = {
+        'Accept': 'application/vnd.github.antiope-preview+json'
+    }
+
+    def _update_attributes(self, suite):
+        # Import here, because a toplevel import causes an import loop
+        from . import pulls
+        from .repos import ShortRepository
```

Comment:
We import the module for pulls, not repos? Is this due to line-length problems?

Reply:
Honestly, this was copy pasta from other places. It could use some normalizing.

Reply:
done